webapp/src/main/java/gov/nist/healthcare/ttt/webapp/common/controller/SVAPValidatorCtrl.java

```diff
@@ -0,0 +1,116 @@
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.apache.logging.log4j.Logger; 
+import org.apache.logging.log4j.LogManager;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import gov.nist.healthcare.ttt.webapp.common.model.exceptionJSON.TTTCustomException;
+
+@Controller
+@RequestMapping("/api/ccdasvap2022")
+public class SVAPValidatorCtrl {
+	String tDir = System.getProperty("java.io.tmpdir");
+	
+	@Value("${ett.mdht.svap.url}")
+	String svapUrl;
+
+	private static Logger logger = LogManager.getLogger(SVAPValidatorCtrl.class.getName());
+
+	@RequestMapping(method = RequestMethod.POST, produces = "application/json")
+	public @ResponseBody String validateSVAP(@RequestBody HashMap<String, String> filePath) throws Exception {
+		if(filePath.containsKey("messageFilePath")) {
+			String messageFilePath = filePath.get("messageFilePath");
+			if(messageFilePath != null && !messageFilePath.equals("")) {
+
+				String validationObjective = "170.315(b)(1)";
+				if(filePath.containsKey("validationObjective")) {
+					validationObjective = filePath.get("validationObjective");
+				}
+				String referenceFileName = "CP_Sample1.pdf";
+				if(filePath.containsKey("referenceFileName")) {
+					referenceFileName = filePath.get("referenceFileName");
+				}
+
+				logger.info("Validating SVAP 2022" + filePath.get("messageFilePath") + " with validation objective " + validationObjective + " and reference filename " + referenceFileName);
+				System.out.println("Validating SVAP 2022" + filePath.get("messageFilePath") + " with validation objective " + validationObjective + " and reference filename " + referenceFileName);
+
+				// Query MDHT war endpoint
+				CloseableHttpClient client = HttpClients.createDefault();
+	    		Path path  = Paths.get(messageFilePath);
+	    		Path normalizedPath =  path.normalize();				
+				String fileName = normalizedPath.toString();
+				File file = new File(fileName);
+				if(!file.exists() || fileName.startsWith("../") || !fileName.startsWith(tDir + File.separator)) {
+					throw new TTTCustomException("0x0050", "Action not supported by API.");
+				}								
+				HttpPost post = new HttpPost(svapUrl);
+				FileBody fileBody = new FileBody(file);
+				//
+				MultipartEntityBuilder builder = MultipartEntityBuilder.create();
+				builder.setMode(HttpMultipartMode.BROWSER_COMPATIBLE);
+				builder.addTextBody("validationObjective", validationObjective);
+				builder.addTextBody("referenceFileName", referenceFileName);
+				builder.addPart("ccdaFile", fileBody);
+				builder.addTextBody("curesUpdate", "true");

```

Comment:
this should be false:
webapp/src/main/java/gov/nist/healthcare/ttt/webapp/common/controller/SVAPValidatorCtrl.java
builder.addTextBody("curesUpdate", "true");